config/eslint-config-carbon/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "eslint-config-carbon",
   "description": "ESLint configuration for Carbon",
-  "version": "2.0.0",
+  "version": "2.1.0",
```

Comment:
```suggestion
  "version": "2.0.0",
```

Reply:
Can we just commit these changes or do we have to do anything else to keep them at `0.0`?